bundles/org.openhab.binding.modbus.kermi/README.md

```diff
@@ -0,0 +1,193 @@
+# Kermi Heat pump
+
+Integrates the X-Center Device (x-center pro) of Kermi Heat pump.
+Kermi X-Center & other attached devices (in progress) are integrated into the Modbus Binding.
+Requirement is contacting Kermi Support to activate Modbus-TCP which can be connected directly by network.
+Older devices (non-Pro ?) were connected by Modbus-RCP - maybe you can try to connect them here using a Modbus-TCP modulator (e.x. from waveshare or similar).
+This binding was tested and developed with Kermi x-change dynamic pro heat pump (build 2023).
+
+See chapter [Thing Configuration](#thing-configuration) how to set them up.
+
+## Supported Things
+
+First you need a TCP-Bridge which establishes the basic connection towards your X-Center device
+
+| Name                     | Thing Type ID | Description                                                                                                                 |
+|--------------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------|
+| Kermi Heat Pump X-Center | kermi-xcenter | Provides (electric) Power values, Flow-information, PV-States, Temperatures and general Information of your Kermi Heat Pump |
+| StorageSystemModule      | kermi-storage | Provides temperatures of your water-storage (drinking water), heating and cooling and details about your heating-circuit    |
```

Comment:
I think i'm missing something here. The readme is suggesting here an there there are two thing type's, but not everywhere. So either the readme has to be updated, or the binding needs to supprt/Thing definition for the kermi-storage